src/store/trails/actions.js

```diff
@@ -193,6 +195,51 @@ export const addBallot = async function ({ commit, state, rootState }, ballot) {
         }
       }
     ]
+    if (ballot.treasurySymbol.symbol === 'VOTE') {
+      togglebal = {
+        account: 'telos.decide',
+        name: 'togglebal',
+        data: {
+          ballot_name: ballotName,
+          setting_name: 'votestake'
+        }
+      }
+      actions.splice(2, 0, togglebal)
+    } else if (!ballot.settings) {
+      togglebal = {
+        account: 'telos.decide',
+        name: 'togglebal',
+        data: {
+          ballot_name: ballotName,
+          setting_name: 'voteliquid'
+        }
+      }
+      actions.splice(2, 0, togglebal)
+    } else if (ballot.settings && ballot.config) {
+      if (ballot.config === 'both') {
+        for (let i of ['voteliquid', 'votestake']) {
+          togglebal = {
+            account: 'telos.decide',
+            name: 'togglebal',
+            data: {
+              ballot_name: ballotName,
+              setting_name: i
+            }
+          }
+          actions.splice(2, 0, togglebal)
+        }
+      } else {
+        togglebal = {
+          account: 'telos.decide',
+          name: 'togglebal',
+          data: {
+            ballot_name: ballotName,
+            setting_name: ballot.config
+          }
+        }
+        actions.splice(2, 0, togglebal)
```

Comment:
Would it be possible to apply a single `actions.splice(2, 0, togglebal)` at line 242? I think that way, we could focus on defining `togglebal` from line 192 up to 242, and then assign it to `actions`. Do you think it makes sense?